pt/lessons/basics/functions.md

```diff
@@ -33,7 +33,7 @@ Como você provavelmente já adivinhou, na versão abreviada nossos parâmetros
 
 ## Pattern matching
 
-Pattern matching não é limitado a apenas variáveis em Elixir, isto pode ser aplicado a assinaturas como veremos nesta seção.
+Pattern matching não é limitado a apenas variáveis em Elixir, isto pode ser aplicado a assinaturas de funções como vemos nesta seção.
```

Comment:
I think it still should be `veremos` instead of `vemos` here.

Reply:
k